scripts/shared/cloud-prepare.sh

```diff
@@ -28,15 +28,19 @@ function check_gateway_exists() {
 }
 
 function prepare_kind() {
-    read -r -a nodes <<< "${cluster_nodes[$cluster]}"
-    local node=${cluster}-${nodes[-1]}
-    kubectl label node "$node" "$GATEWAY_LABEL" --overwrite
-
-    if [[ "$AIR_GAPPED" = true ]]; then
-        local pub_ip
-        pub_ip=$(kubectl get nodes "$node" -o jsonpath="{.status.addresses[0].address}")
-        kubectl annotate node "$node" gateway.submariner.io/public-ip=ipv4:"$pub_ip"
-    fi
+    local gw_count="${cluster_gateways[$cluster]:-1}"
+
+    readarray -t nodes < <(kubectl get nodes --no-headers -o custom-columns=NAME:".metadata.name" | tac)
```

Comment:
```suggestion
    readarray -d' ' nodes < <(kubectl get nodes -o jsonpath="{.items[*].metadata.name}")
```

Or if you insist on reversing the order (even though it doesn't matter):
```suggestion
    readarray -t nodes < <(kubectl get nodes -o yaml | yq '.items[].metadata.name' | sort -r)
```

Reply:
Updated.